server.js

```diff
@@ -42,6 +42,9 @@ app.prepare().then(() => {
   server.get('/index.html', (req, res) => {
     res.redirect(301, '/')
   })
+  server.get('/crew', (req, res) => {
+    res.redirect(301, '/')
```

Comment:
let's do a `302` to https://www.republik.ch/impressum

Not a 301 in this case because eventually there might be a page again – browsers and Google should not cache the redirect.